external/kafka-assembly/pom.xml

```diff
@@ -43,6 +43,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
+      <artifactId>spark-streaming-kafka-newapi_${scala.binary.version}</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
```

Comment:
Mark, i re-read your description above and i think you are suggesting that we support old consumer API's only with 0.9 brokers, in which case we dont need the above.